install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -8,3 +8,6 @@ metadata:
     include.release.openshift.io/hypershift: "true"
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
+data:
+  ack-4.18-kube-1.32-api-removals-in-4.19: |-
+      Kubernetes 1.32 and therefore OpenShift 4.19 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/7112216 for details and instructions.
```

Comment:
Hi @wking  as per the doc link https://access.redhat.com/articles/7112216,
```
Per the [Kubernetes documentation](https://access.redhat.com/bounce/?externalURL=https%3A%2F%2Fkubernetes.io%2Fdocs%2Freference%2Fusing-api%2Fdeprecation-guide%2F%23v1-29), Kubernetes 1.32 removed several deprecated APIs. OpenShift Container Platform 4.19 plans to use Kubernetes 1.32.
Unlike past releases of OpenShift Container Platform that introduced Kubernetes API removals, OpenShift Container Platform 4.19 does not require an administrator acknowledgment before upgrading to it.
```
bit confused over the statements, and i dont see in doc that the acknowledgement is required.

Reply:
Nice catch!  It's @bergerhoffer 's KCS, and she's said she can have it updated and published by Wednesday the 30th, when the next 4.18.z that might consume this will be built.  So I think we can merge as it stands, and have the KCS updated to call out the need for an ack by the time customers might be running clusters with the gate.

Reply:
KCS is updated to reflect that there is actually going to be an admin-ack this time. Once this PR merges and we know what z-stream it'll be in, I'll update the KCS with that value and publish.